package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "media-chrome",
-  "version": "0.3.2",
+  "version": "0.3.0",
```

Comment:
Don't manually bump the version. I use [np](https://www.npmjs.com/package/np) which handles bumping, tagging, and github releases, so a manual bump messes that up.